service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,7 +4,84 @@ include::modules/ossm-document-attributes.adoc[]
 :context: updating-ossm
 toc::[]
 
-If you are updating from {ProductName} 1.0 to 1.1, you must update the `ServiceMeshControlPlane` resource to update the control plane components to the new version. 
+
+== Manual updates required by version 1.1.4
+
+The fix for https://bugzilla.redhat.com/show_bug.cgi?id=1844254[CVE-2020-8663]`: envoy: Resource exhaustion when accepting too many connections` added a configurable limit on downstream connections. The configuration option for this limit must be configured to mitigate this vulnerability.
```

Comment:
Should the BZ link be prepended with `link:` here? That's how we normally format external links, though seems to work without it.

Reply:
Copy/paste error, thanks!